packages/vue/src/components/organisms/SfTabs/_internal/SfTab.spec.js

```diff
@@ -0,0 +1,8 @@
+import { shallowMount } from "@vue/test-utils";
+import SfTab from "./SfTab.vue";
+describe("SfTab.vue", () => {
+  it("renders a component", () => {
+    const component = shallowMount(SfTab);
+    expect(component.contains(".sf-tabs__title")).toBe(true);
```

Comment:
This won't work too good - what about `sf-tabs__content` ? You can solve it two ways:
1)  Add 'sf-tabs__tab' class to the wrapper component
2) ` mount` instead of  `shallowMount` to check if it contains `Fragment`: https://vue-test-utils.vuejs.org/api/wrapper/contains.html

Reply:
Can you throw more light on how to check if it contains Fragment? The docs didn't say much. 
Is this right?
```
import { mount } from '@vue/test-utils'
import SfTab from "./SfTab.vue";
import Fragment from './SfTab.vue'

describe("SfTab.vue", () => {
  it("renders a component", () => {
    const component = mount(SfTab);
    expect(wrapper.contains(Fragment)).toBe(true)
  });
});
```

Reply:
component is the wrapper :)This should work fine: 

`import { mount } from '@vue/test-utils'
import SfTab from "./SfTab.vue";
import Fragment from './SfTab.vue'

describe("SfTab.vue", () => {
  it("renders a component", () => {
    const component = mount(SfTab);
    expect(wrapper.contains(Fragment)).toBe(true)
  });
});`

Reply:
or simply `<Fragment class="sf-tabs__tab">` and check if it renders class